recipes/gstreamer/all/conandata.yml

```
@@ -1,4 +1,10 @@
 sources:
+  "1.22.3":
+    url: "https://gstreamer.freedesktop.org/src/gstreamer/gstreamer-1.22.3.tar.xz"
```

```suggestion
    url: "https://gitlab.freedesktop.org/gstreamer/gstreamer/-/archive/1.22.3/gstreamer-1.22.3.tar.gz"
```
@uilianries suggest that they have had issues with the downloads from gstreamer.freedesktop, and gitlab downloads have been more reliable

---

GitLab isn't reliable, as well as GitHub. they auto-generate these archives with `git archive` format. and sometimes they can upgrade git version, or change `git config`, after that sha256 no longer matches previously generated archive. it's annoying, but it already happened in the past multiple times.